pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -223,7 +226,10 @@ func (s CPUSet) ToSliceInt64() []int64 {
 // ToSliceNoSortInt64 returns a slice of int64 that contains all elements from
 // this set.
 func (s CPUSet) ToSliceNoSortInt64() []int64 {
-	var result []int64
+	if s.IsEmpty() {
+		return nil
+	}
```

Comment:
Is this judgment necessary? When the for iteration is empty, can it not be returned correctly?

Reply:
This is to keep the logic consistent with the previous one. Uninitialized slice can be returned